### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by yalh76**
## Problem

- *Level 6*

## Solution

- *Apply example_ynh*

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

File: scripts/_common.sh [hunk 1]
```diff
@@ -5,26 +5,12 @@
 #=================================================
 
 # dependencies used by the app
-pkg_dependencies=(
-    python3 virtualenv
-    gettext libzip-dev libssl-dev
-    mariadb-server default-libmysqlclient-dev
-    uwsgi uwsgi-plugin-python3 python3-uwsgidecorators
-)
+pkg_dependencies="python3 virtualenv gettext libzip-dev libssl-dev mariadb-server default-libmysqlclient-dev uwsgi uwsgi-plugin-python3 python3-uwsgidecorators"
```
Comment (Salamandar) [inline scripts/_common.sh:8]: I should make a pull request to the example repository to change this.

**Comment by yalh76** [comment]
!testme

**Comment by Salamandar** [comment]
@yalh76 There's no documentation about ynh_clean_check_starting. It seems like an internal helper that shouldn't be used in app code ?

**Comment by yalh76** [comment]
> ynh_clean_check_starting

My bad, it's an internal helper since two years ...

**Comment by yalh76** [comment]
!testme

File: scripts/change_url [hunk 2]
```diff
@@ -103,17 +104,19 @@ then
...
 fi
 
 #=================================================
 # SPECIFIC MODIFICATIONS
 #=================================================
-# Reedit configuration files
+# UPDATE A CONFIG FILE
 #=================================================
-ynh_script_progression --message="Updating app's configuration..." --weight=1
+ynh_script_progression --message="Updating a configuration file..." --weight=1
```
Comment (Salamandar) [inline scripts/change_url:115]: Why this change ? I mean for the user "a" is a bit weird, while "the" or "app's" might be clearer.

**Review by Salamandar — APPROVED** [review]

File: scripts/_common.sh (hunk 1 above)
Comment (yalh76) [reply scripts/_common.sh:8]: If you want, but having a multiline dependencies make more difficult for package_linter to parse the file

File: scripts/change_url (hunk 2 above)
Comment (yalh76) [reply scripts/change_url:115]: because it's written like that in the other files
